### ISSUE TITLE
Merge master to release-4.12 branch and merge release-4.13 to master 

### ISSUE THREAD

**Issue by praveenkumar** [crc-org/snc#724]
Since 4.13 is released so it is time to do the following.

- merge current master to release-4.12 branch
- merge release-4.13 to master branch

**Comment by praveenkumar** [comment]
- #723 is created for first one and once it is merged then we will create another PR for second task.

**Comment by cfergeau** [comment]
> Since 4.13 is released so it is time to do the following.

The time to do this is not strictly when 4.13 is released, but when our team decides to move its focus to 4.13, and expects to no longer release new 4.12 bundles.

**Comment by praveenkumar** [comment]
> The time to do this is not strictly when 4.13 is released, but when our team decides to move its focus to 4.13, and expects to no longer release new 4.12 bundles.

we can still generate 4.12 bundles with release-4.12 branch.

**Comment by praveenkumar** [comment]
both are done.
